src/NHSD.BuyingCatalogue.Identity.Common/Email/MimeKitExtensions.cs

```diff
@@ -27,10 +28,13 @@ internal static MimeMessage AsMimeMessage(this EmailMessage emailMessage)
 
             var message = new MimeMessage
             {
-                Subject = emailMessage.Subject ?? string.Empty,
                 Body = bodyBuilder.ToMessageBody(),
             };
 
+            message.Subject = string.IsNullOrWhiteSpace(emailSubjectPrefix) && emailMessage.Subject != null
```

Comment:
As mentioned on a call. Please review this section of code. The original request was to trim any whitespace from the subject of the email. What happens here if `email.Subject` is equal to an empty string.

Reply:
👍